modules/virt-attaching-vm-to-primary-udn.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-attaching-vm-to-primary-udn_{context}"]                                  
+= Attaching a virtual machine to the primary user-defined network
+
+You can connect a virtual machine (VM) to the primary user-defined network (UDN) by requesting the pod network attachment, and configuring the interface binding.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Edit the `VirtualMachine` manifest to add the UDN interface details, as in the following example:
++
+.Example `VirtualMachine` manifest
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: example-vm
+  namespace: my-namespace # <1>
+spec:
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+            - name: udn-l2-net # <2>
+              binding: 
+                name: l2bridge # <3>
+# ...
+      networks:
+      - name: udn-l2-net # <4>
+        pod: {}
+# ...
+----
+<1> The namespace in which the VM is located. This value must match the namespace in which the UDN is defined.
+<2> The name of the user-defined network interface.
+<3> The name of the binding plugin that is used to connect the interface to the VM. The required value is `l2bridge`.
```

Comment:
nit: Please consider following existing docs https://docs.openshift.com/container-platform/4.17/virt/vm_networking/virt-connecting-vm-to-linux-bridge.html#virt-attaching-vm-secondary-network-cli_virt-connecting-vm-to-linux-bridge